src/main/webapp/dev/js/common/instructorFeedbackResults.js

```diff
@@ -149,11 +149,41 @@ function replaceButtonHtmlAndTooltipText(button, from, to) {
 function expandOrCollapsePanels(expandCollapseButton, panels) {
     const STRING_EXPAND = 'Expand';
     const STRING_COLLAPSE = 'Collapse';
+    const expandCollapseStudentsButton = 'a[id^="collapse-panels-button-section-"]';
+    const expandCollapseTeamButton = 'a[id^="collapse-panels-button-team-"]';
+    const isButtonInExpandMode = $(expandCollapseButton).html().trim().startsWith(STRING_EXPAND);
+    const isCollapsePanelsButton = !panels;
+    /*
+     * When collapsing expandCollapseButton, all panels buttons under it,
+     * (expandCollapseTeamButton and  expandCollapseStudentButton) should be in collapsed state,
```

Comment:
Actually, what are the brackets for?

Reply:
To clarify which panel buttons are we talking about. 
Well on another thought, it's not really necessary. I'll modify it.